crates/swc/tests/errors/lints/use-is-nan/js/default/output.swc-stderr

```diff
@@ -0,0 +1,72 @@
+
+  x Use the isNaN function to compare with NaN
+   ,----
+ 2 | if (foo == NaN) {}
```

Comment:
@kdy1 seems like lines numeration is broken.

https://github.com/swc-project/swc/pull/4080/files#diff-1e3f349d2c56fc46384b18d56f835ba0ee7e04f1f3f32c489b58b6689ed9b5f7R1

Reply:
Yeah, I'll fix it